lib/serializables/log-event.js

```diff
@@ -51,9 +51,6 @@ function LogEventFactory (
             host: {
                 type: 'string'
             },
-            module: {
```

Comment:
@sunnyqianzhang @yyscamper I also think we should keep the field `module`, which provides more accurate info.